test/util/test_runners/integrations_tests_runner.py

```diff
@@ -90,41 +90,15 @@ def start_env(self):
         """
 
         path_provider = TestRunnerPathProvider(file_helper_dir=self.runner_args.files_folder)
-        self.__create_conjurrc_file()
-        self.__create_netrc_file()
+        # self.__create_conjurrc_file()
```

Comment:
we don't need these at all?

Reply:
if not just remove them. Why did we need them in the first place?

Reply:
some tests required conjurrc and netrc file to work with. i think that we improved our integration tests init and this is no longer required.
If you could run it in your env just to make sure It's not the leftovers that make it pass on my side it will be great

Reply:
@mbenita-Cyberark as discussed all things related to the integration test as process should be a separate PR @mbenita-Cyberark

Reply:
@mbenita-Cyberark